GoogleScholarPlugin.php

```diff
@@ -115,8 +115,13 @@ public function submissionView($hookName, $args)
         $authors = $publication->getData('authors');
         foreach ($authors as $i => $author) {
             $templateMgr->addHeader('googleScholarAuthor' . $i, '<meta name="citation_author" content="' . htmlspecialchars($author->getFullName(false, false, $publicationLocale)) . '"/>');
-            if ($affiliation = $author->getLocalizedData('affiliation', $publicationLocale)) {
-                $templateMgr->addHeader('googleScholarAuthor' . $i . 'Affiliation', '<meta name="citation_author_institution" content="' . htmlspecialchars($affiliation) . '"/>');
+            foreach($author->getLocalizedAffiliations() as $affiliation) {
```

Comment:
Localized Affiliations would be in the users UI, which we do not want -- we would like the affiliations in the publication/submission locale :-)

Reply:
Google Scholar confirmed:

> It would be best to repeat citation_author_institution directly after citation_author. For example:
> ```
> <meta name="citation_author" content="Joanne E. Mortimer" />
> <meta name="citation_author_institution" content="Department of Medical Oncology and Experimental Therapeutics, City of Hope, Duarte, California" />
> <meta name="citation_author_institution" content="Department of Radiology, Stanford University Medical Center, Stanford, California" />
> ```
> ROR IDs would not be included. It would be best to use institution names only.

Reply:
Not ideal, but we can do it. I'll ask them what their expectations are for the language of the institution.

Reply:
Changed to $author->getLocalizedAffiliations($publicationLocale)